lib/metadataTypes/definitions/Interaction.definition.js

```diff
@@ -388,15 +388,15 @@ module.exports = {
         },
         version: {
             isCreateable: false,
-            isUpdateable: false,
+            isUpdateable: true,
```

Comment:
explain?

Reply:
thats the journey version you want to update

Reply:
if we do not send it during an update, the API returns the error "Interaction not found" (did test this one in particular)